extensions-core/druid-basic-security/src/main/java/org/apache/druid/security/basic/authentication/db/cache/CoordinatorBasicAuthenticatorCacheNotifier.java

```diff
@@ -53,24 +53,32 @@ public CoordinatorBasicAuthenticatorCacheNotifier(
       @EscalatedClient HttpClient httpClient
   )
   {
-    cacheNotifier = new CommonCacheNotifier(
+    userCacheNotifier = new CommonCacheNotifier(
         initAuthenticatorConfigMap(authenticatorMapper),
```

Comment:
should this be initAuthenticatorUserMap instead of configMap ?

Reply:
No, initAuthenticatorConfigMap has nothing to do with user/s per say. initAuthenticatorConfigMap is unchanged and just returns the map of authenticator prefix and authenticator configuration (BasicAuthDBConfig) that the CommonCacheNotifier (userCacheNotifier) needs too reference configuration properties like isEnableCacheNotifications